portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSet.testcase

```diff
@@ -520,6 +878,88 @@ definition {
 		}
 	}
 
+	@description = "LPS-185990 Confirm that when creating a filter within the view, there is a search bar field to preselect values"
+	@priority = 5
+	test CanAssertTheSearchBarField {
+		task ("Given the creation of a new data set") {
+			DataSetAdmin.goToDataSetAdminPage();
+
+			DataSetAdmin.createDataSetEntryViaAPI(
+				dataSetName = "Data Set Test",
+				restApplication = "/data-set-manager/fields",
+				restEndpoint = "/",
+				restSchema = "FDSField");
+
+			DataSetAdmin.createFDSViewViaAPI(
+				dataSetName = "Data Set Test",
+				dataSetViewDescription = "FDSViewDescription",
+				key_dataSetViewNameList = "View Test");
+
+			DataSetAdmin.goToSpecificViewPage(
+				dataSetName = "Data Set Test",
+				key_viewName = "View Test");
+		}
+
+		task ("And creating a picklist and Items bike,car,bus") {
+			Picklist.addPicklistViaAPI(picklistName = "Transport");
+
+			Picklist.addPicklistItemViaAPI(
+				itemKey = "bike",
+				itemName = "bike",
+				picklistName = "Transport");
+
+			Picklist.addPicklistItemViaAPI(
+				itemKey = "car",
+				itemName = "car",
+				picklistName = "Transport");
+
+			Picklist.addPicklistItemViaAPI(
+				itemKey = "bus",
+				itemName = "bus",
+				picklistName = "Transport");
+		}
+
+		task ("When the user creates a new filter //Name: Filter test //Filter by: creator //Source option: Transport the selection is multiple") {
+			DataSetAdmin.goToTab(tabName = "Filters");
+
+			LexiconEntry.gotoAdd();
+
+			Click(
+				key_fieldName = "Selection",
+				locator1 = "FormViewBuilder#OBJECT_FIELD_BUTTON_DROPDOWN");
```

Comment:
Why are you using a FormViewBuilder path? Please use a generic dropdown button path